dotcom-rendering/src/server/prod-server.ts

```diff
@@ -103,6 +105,21 @@ export const prodServer = () => {
 		},
 	);
 
+	app.get(
+		'/Front',
+		logRenderTime,
+		getContentFromURLMiddleware,
+		async (req: Request, res: Response) => {
+			// Eg. http://localhost:9000/AMPArticle?url=https://www.theguardian.com/commentisfree/...
+			try {
+				return renderAMPArticle(req, res);
```

Comment:
I'm guessing this isn't supposed to be renderAMPArticle?

Reply:
At least one of us isn’t sleeping at the wheel!